api/router.go

```diff
@@ -69,6 +69,9 @@ func NewRouter(s *Server) *mux.Router {
 		"recurringUpdate":    s.VolumeRecurringUpdate,
 		"updateReplicaCount": s.VolumeUpdateReplicaCount,
 
+		"createAssociatedPV":  s.AssociatedPVCreate,
```

Comment:
`pvCreate` and `pvcCreate`, also add to the end of the list.

Notice the `action` always start with the object first, then action, e.g. `snapshotCreate`, `replicaRemove`.